src/metabase/models/notification.clj

```diff
@@ -89,14 +90,15 @@
 (def ^:private Notification
   [:merge
    [:map
-    [:payload_type (into [:enum] notification-types)]]
+    [:payload_type (apply ms/enum-decode-keyword notification-types)]]
```

Comment:
Why `apply` everywhere if it's your function, maybe it should just accept a list? :)